scrapy/core/scraper.py

```diff
@@ -18,6 +19,11 @@
 from scrapy.core.spidermw import SpiderMiddlewareManager
 from scrapy.utils.request import referer_str
 
+
+if six.PY3:
```

Comment:
As @kmike mentioned on other pull requests, ``if not six.PY2:`` would be more future-proof. Same a few lines below.